src/Fantomas.CoreGlobalTool.Tests/TestHelpers.fs

```diff
@@ -0,0 +1,39 @@
+module Fantomas.CoreGlobalTool.Tests.TestHelpers
+
+open System
+open System.Diagnostics
+open System.IO
+
+type TemporaryFileCodeSample internal (codeSnippet: string) =
+    let filename = Path.Join(Path.GetTempPath(), Guid.NewGuid().ToString() + ".fs")
+    do File.WriteAllText(filename, codeSnippet)
+
+    member _.Filename: string = filename
+    interface IDisposable with
+        member this.Dispose(): unit = File.Delete(filename)
+
+let private runFantomasTool arguments =
+    let pwd = Path.GetDirectoryName(typeof<TemporaryFileCodeSample>.Assembly.Location)
+    let configuration =
+        #if DEBUG
+        "Debug"
+        #else
+        "Release"
+        #endif
+
+    let fantomasDll =
+        Path.Combine
+            (pwd, "..", "..", "..", "..", "Fantomas.CoreGlobalTool", "bin", configuration, "netcoreapp3.1",
+             "fantomas-tool.dll")
+
+    use p = new Process()
+    p.StartInfo.UseShellExecute <- false
+    p.StartInfo.FileName <- @"dotnet"
+    p.StartInfo.Arguments <- sprintf "%s %s" fantomasDll arguments
+    p.Start() |> ignore
+    p.WaitForExit()
+    p.ExitCode
+
+let checkCode file =
```

Comment:
What about moving `checkCode` to the corresponding test file where it is used and exposing the `runFantomasTool` (which looks pretty solid to expose as a test helper, kudos for that) for other tests?

Reply:
You're right that makes more sense.
I'd typically do things like these when the next test suite would pop up.